docs/LorisPHPMD.xml

```diff
@@ -16,4 +16,9 @@
 			about them not being used -->
 		<exclude name="UnusedFormalParameter" />
 	</rule>
+
+    <rule ref="rulesets/design.xml">
+        <exclude name="CouplingBetweenObjects" />
+        <exclude name="ExitExpression" />
```

Comment:
Going from the list of design rules at https://phpmd.org/rules/design.html, I think the only two we should be enforcing are "ExitExpression", "DevelopmentCodeFragment", and "EvalExpression" since they're things that people are already looking for in code review and it saves times to have Travis do them instead.

I don't think the others should be enforced, because they'll just cause people to be frustrated when they can't figure out Travis errors because of some arbitrary tool developer's opinion.

Reply:
Okay, I'll add the others to the excluded list. For now I think it would be best to continue to exclude `ExitExpression` because it caused a bunch of errors. These errors can be solved in another pull request to keep things simple.

Reply:
Okay